app/views/work_orders/cancellations/new.erb

```diff
@@ -0,0 +1,72 @@
+<% content_for :title, "Cancel Works Order: #{@work_order.reference}" %>
+<% content_for :back_button do %>
+  <%= link_to 'Back', work_order_path(@work_order.reference), class: 'govuk-back-link' %>
+<% end %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-l">
+        Cancel repair
+      </span>
+      <h1 class="govuk-heading-l">
+        Works order: <%= @work_order.reference %>
+      </h1>
+
+      <table class="govuk-table">
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header" scope="col">
+            Property
+          </th>
+          <td class="govuk-table__cell">
+            <%= @work_order.property.description.singularize %>,
+            <%= @work_order.property.first_line_of_address %>
+          </td>
+        </tr>
+
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header" scope="col">
+            Trade
+          </th>
+          <td class="govuk-table__cell">
+            <%= @work_order.trade %>
+          </td>
+        </tr>
+
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header" scope="col">
+            Description
+          </th>
+          <td class="govuk-table__cell">
+            <%= @work_order.problem_description %>
+          </td>
+        </tr>
+      </table>
+
+      <!--
+      <h2 class="govuk-heading-m">
+        Reason to cancel
+      </h2>
+      -->
+
+      <%= govuk_form_with(
+        model: @cancellation,
+        url: work_order_cancellation_path(@work_order.reference),
+        local: true
+      ) do |form|
+      %>
+
+        <%= form.form_group :reason do %>
+          <%= form.label :reason, "Reason to cancel" %>
```

Comment:
I guess this is a design thing, but this isn't great English. Should be something like "Reason for cancelling"

Reply:
I'll tell Boris

Reply:
he agrees :sparkles: